src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.csproj

```diff
@@ -143,4 +147,12 @@
   <!-- SDK bottom import -->
   <Import Project="Sdk.targets" Sdk="MSBuild.Sdk.Extras" Condition=" '$(OS)' == 'Windows_NT' " />
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" Condition=" '$(OS)' != 'Windows_NT' " />
+  <ItemGroup>
```

Comment:
dunno what this is about, this project is somehow busted, every time I add files the get removed from compilation. Removing this.

Reply:
if we want to keep the version without reflection we need to insert a static source or a "resource"? but I don't know if at runtime we can get it also in aot mode.

Reply:
That file is not added to the build if not with manual *.cs at least it wasn't working without custom source addition.